src/components/invite-dialog/index.test.tsx

```diff
@@ -85,14 +91,14 @@ describe('InviteDialog', () => {
     expect(onClose).toHaveBeenCalled();
   });
 
-  it('displays text in green if no invites remaining', function () {
+  it('displays text in green if invites remaining', function () {
```

Comment:
We should wait for Ethan to confirm this is what he wants. The designs have the text green when the invites remaining hits zero.

Reply:
Will do :) thanks!